sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
+* Fixed an issue where initial `targetBatchSize` is larger than `maxBatchSize` configured in `CosmosBulkExecutionOptions` - See[39500](https://github.com/Azure/azure-sdk-for-java/pull/39500)
```

Comment:
NIT - i would  classify this as "Other" - you can argue either way what the expected way should be. And honoring the config is not really a bug?

Reply:
updated